Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -477,7 +477,7 @@ float VideoInterfaceManager::GetAspectRatio() const
   // signal (which would be 4:3)
 
   // This function only deals with standard aspect ratios. For widescreen aspect ratios,
-  // multiply the result by 1.33333..
+  // multiply the result by 1.33333 (the ratio between 16:9 and 4:3)
```

Comment:
The `..` was intentional here to suggest 'infinitely repeating', I'm pretty sure.

Reply:
Ah 😅. I'll fix that.